website/content/tutorials/3-constraints-expect/_index.md

```diff
@@ -0,0 +1,1511 @@
+---
+title: "Complex Constraints: expect"
+description: ""
+---
+
+# Complex Constraints
+
+## Introduction
+
+In [the previous tutorial](/tutorials/2-constraints/), we refined a computer model and learned how to restrict or recommend preferred values for fields and flags with allowed-values constraints. But what do we if our use case requires values to have a consistent style or structure, but there is no predetermined list of values our stakeholders know upfront? Additionally, what do we do if we have such requirements for field and flag values, but they must also correlate to related field or flag values in the same document instance, or maybe multiple related instances the software processes at the same time?
```

Comment:
"Additionally, what do we do if we have such requirements for field and flag values" What does "such" reference?

Reply:
Such is short for requirements where "there is no predetermined list of values our stakeholders know upfront" from the previous sentence.

Reply:
I don't think it is clear in the text what "such" references. Can you adjust the text to make this more clear?

Reply:
Sure thing.

Reply:
I tried to handle this and some other final cleanup in https://github.com/usnistgov/metaschema/pull/530/commits/7719523f084f9dd055f2676d6a86881f9f77e67e.